sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Microsoft.Azure.Management.Attestation.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>Attestation.Tests</PackageId>
+    <Version>1.0.0</Version>
+    <Description>Attestation.Tests Class library</Description>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.Azure.Management.Attestation.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.ResourceManager">
+      <HintPath>..\..\..\..\..\..\Users\shlei\.nuget\packages\microsoft.azure.management.resourcemanager\1.9.1-preview\lib\net452\Microsoft.Azure.Management.ResourceManager.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="SessionRecords\Attestation.Tests.ScenarioTests.AttestationOperationsTests\" />
+  </ItemGroup>
```

Comment:
@shleiAmy , no need to add line 31-33 since new sdk style project system will automatically include any sub folders.

Reply:
removed 
  <ItemGroup>
    <Folder Include="SessionRecords\Attestation.Tests.ScenarioTests.AttestationOperationsTests\" />
  </ItemGroup>